main.py

```diff
@@ -177,6 +190,24 @@ def broadcast_block(self, block_hash: str, block: Block):
             self.ez_send(peer, blockMessage)
 
 
+    def send_peers(self) -> None:
+        peers = self.get_peers()
+        peerMessage = PeersMessage(self.my_peer.mid)
+
+        for peer in peers:
+            self.ez_send(peer, peerMessage)    
+
+
+    @lazy_wrapper(PeersMessage)
+    def receive_peers(self, peer: Peer, payload: PeersMessage) -> None:
```

Comment:
I think that we need to add a condition that only if we don't know the mid add to the list and braodcast to the other peers